src/routers/suggestionRoutes.ts

```diff
@@ -0,0 +1,15 @@
+import express from 'express';
+import { UserModel } from '../models/UserModel';
+import { UserService } from '../services/UserService';
+
+const router = express.Router();
+const UserServiceInstance = new UserService(UserModel);
```

Comment:
seems like you have 2 equal instances of `UserService`, I suggest creating a singleton of `UserService` or keep `userRoutes`  and `suggestionRoutes` in the same file.

Reply:
Fixed